src/config.ts

```diff
@@ -78,6 +78,34 @@ export const SCENE_TILE_PROVIDERS = [
   },
 ]
 
+export const TOUR = {
+  algorithm: ':first-child',
+  /*
+  apiKeyInstructions: `
+    To get an API key, ask the experts at <a
+      href="https://rocketchat.gs.mil/channel/planetdas_askanexpert"
+      target="_blank"
+    >Rocket.Chat #planetdas_askanexpert</a>.
+  `,
+  */
+  apiKeyInstructions: `
+    To get an API key you must have a Planet account.  For more information see <a
+      href="https://support.planet.com/hc/en-us/articles/212318178-What-is-my-API-key-"
+      target="_blank"
+    >What Is My API Key?</a>
+  `,
+  basemap: 'OSM',
+  bbox: [-85, 19.7, -74, 23.4],
```

Comment:
I assume everyone will have access to this location.  I know sometimes I'll get an error from planet saying something to the effect of "Your api key doesn't give you access to this area".  I assume everyone would be able to search that area but it's a potential problem I guess.

Reply:
That message is _always_ a red herring, since it is hard-coded in the bf-ui code to display for any error coming back from bf-api for creating a job.